libdrgn/Makefile.am

```diff
@@ -87,8 +87,8 @@ libdrgnimpl_la_SOURCES = $(ARCH_DEFS:.defs=.c) \
 			 vector.c \
 			 vector.h
 
-libdrgnimpl_la_CFLAGS = -fvisibility=hidden $(OPENMP_CFLAGS) $(elfutils_CFLAGS)
-libdrgnimpl_la_LIBADD = $(OPENMP_LIBS) $(elfutils_LIBS)
+libdrgnimpl_la_CFLAGS = -fvisibility=hidden $(ASAN_CFLAGS) $(OPENMP_CFLAGS) $(elfutils_CFLAGS)
```

Comment:
I think we also want the Python bindings built with ASAN, as well as any example programs. So maybe we should add the  ASAN `CFLAGS` and `LDFLAGS` to `AM_CFLAGS` and `AM_LDFLAGS` instead, and then add `AM_CFLAGS` and `AM_LDFLAGS` to all of our current per-target `CFLAGS` and `LDFLAGS`.

Reply:
I'm not terribly familiar with autotools, but I believe I have understood your intentions correctly and implemented them in my latest force-push.